build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException.cs

```diff
@@ -2,41 +2,18 @@
 
 namespace Microsoft.DotNet.Cli.Build.Framework
 {
-    public class BuildFailureException : Exception
+    public partial class BuildFailureException : Exception
```

Comment:
Explained somewhat above. 

I am source including these files in the test utilities to prevent source duplication and yet another command, dotnet, etc.. class. I broke apart these into partial classes so I could source include less files which were non-essential. If I left this in one file, I'd have to include BuildTarget.cs, and whatever it depends on.